### ISSUE TITLE
Missing a closing node

### ISSUE THREAD

**Issue by lennybacon** [dotnet/docs#35554]
Instead of:

```xml
<ProjectReference Include="..\CS0433_1\CS0433_1.csproj">  
  <Aliases>CustomTypes</Aliases>  
<ProjectReference Include="..\CS0433_2\CS0433_2.csproj" /> 
```

it should be:

```xml
<ProjectReference Include="..\CS0433_1\CS0433_1.csproj">  
  <Aliases>CustomTypes</Aliases>  
</ProjectReference>
<ProjectReference Include="..\CS0433_2\CS0433_2.csproj" /> 
```

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: cc727a99-200d-d34b-a2db-72f336f6f326
* Version Independent ID: fa87a308-62df-2b4c-1685-2ff9372b5c46
* Content: [Compiler Error CS0433](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs0433)
* Content Source: [docs/csharp/language-reference/compiler-messages/cs0433.md](https://github.com/dotnet/docs/blob/main/docs/csharp/language-reference/compiler-messages/cs0433.md)
* Product: **dotnet-csharp**
* Technology: **csharp-diagnostics**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

**Comment by nayangoyal** [comment]
Hii, Can I work on this issue? @lennybacon

**Comment by postmeback** [comment]
As the PR is already merged, I think it should be good to close.

**Comment by lennybacon** [comment]
Thanks!

**Comment by Kissaki** [comment]
This was fixed in 3b61eeced55fcfe739b321a92ffb991216947ff3. Please close the ticket.

**Comment by lennybacon** [comment]
Thx
